src/main/java/com/google/cloud/genomics/dataflow/pipelines/CountReads.java

```diff
@@ -193,4 +223,19 @@ public SearchReadsRequest apply(Contig shard) {
                   contigs.iterator().next())));
     }
   }
+
+  private static Iterable<Contig> parseContigs(String c) {
```

Comment:
You're right, I should have used Contig.parseContigsFromCommandLine. Let me
fix that.

On Thu, Apr 16, 2015 at 3:31 PM, Nicole Deflaux <notifications@github.com>
wrote:

> In
> src/main/java/com/google/cloud/genomics/dataflow/pipelines/CountReads.java
> <https://github.com/googlegenomics/dataflow-java/pull/55#discussion_r28556101>
> :
>
> > @@ -193,4 +223,19 @@ public SearchReadsRequest apply(Contig shard) {
> >                    contigs.iterator().next())));
> >      }
> >    }
> > +
> > +  private static Iterable<Contig> parseContigs(String c) {
>
> This method can go away. There is a version of it in utils-java.
>
> See
> https://github.com/googlegenomics/dataflow-java/blob/master/src/main/java/com/google/cloud/genomics/dataflow/utils/GenomicsDatasetOptions.java#L50
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/googlegenomics/dataflow-java/pull/55/files#r28556101>.
>